interface/src/index.html

```diff
@@ -15,12 +15,29 @@
     <!-- font awesome from BootstrapCDN -->
     <link href="http://netdna.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.css" rel="stylesheet">
 
+
+    <!-- Required CSS for ui-slider -->
+    <link rel="stylesheet"
+          href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/smoothness/jquery-ui.css">
+
     <!-- compiled CSS --><% styles.forEach( function ( file ) { %>
     <link rel="stylesheet" type="text/css" href="/static/<%= file %>" /><% }); %>
 
+    <!-- Required js for ui-slider
+         FIXME: Why adding these two libraries in build.config.js does not work?
```

Comment:
Thank you! I hate asking you to change code that is working, but I really do feel that it would become a problem for maintenance down the road.

One other note about jQuery and Angular that I can't find now but saw in my reading is that I think you only get to include it once, so if one of your modules requires version A.8 of jQuery and another requires version A.7 but isn't yet compatible with A.8 then you are really stuck.